src/Common/src/Interop/Windows/user32/Interop.FindWindow.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class User32
+    {
+        [DllImport(Libraries.User32, CharSet=CharSet.Auto)]
```

Comment:
Interop guidelines say

>> Explicitly specify the CharSet as Ansi or Unicode when the signature has a string.

https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/interop-guidelines.md#definition

And if there's no string I guess you can remove it.

Reply:
Ok, I'll do a pass over these.